srcpkgs/mesa/template

```diff
@@ -1,19 +1,20 @@
 # Template file for 'mesa'
 pkgname=mesa
-version=20.2.3
+version=20.3.1
 revision=1
 wrksrc="mesa-${version}"
 build_style=meson
 configure_args="-Dglvnd=true -Dshared-glapi=true -Dgbm=true -Degl=true
  -Dosmesa=gallium -Dgles1=true -Dgles2=true -Dglx=dri -Ddri3=true
- -Dlmsensors=true -Dplatforms=x11,drm,$(vopt_if wayland wayland,)surfaceless
- -Dllvm=true -Db_lto=false -Dcpp_std=gnu++14"
+ -Dlmsensors=true -Dplatforms=x11$(vopt_if wayland ',wayland')
+ -Dllvm=true -Db_lto=false -Dcpp_std=gnu++14
+ -Ddri-drivers-path=/usr/lib${XBPS_TARGET_WORDSIZE}"
```

Comment:
shouldn't this be `/usr/lib${XBPS_TARGET_WORDSIZE}/dri`?

Reply:
Also remove the block
```
case "$XBPS_TARGET_MACHINE" in
	i686) configure_args+=" -Ddri-drivers-path=/usr/lib32/dri";;
esac
```

Reply:
yes, sorry, i accidentally wrote the wrong argument in the previous PR

Reply:
xorg also uses this path (via `/usr/lib/pkgconfig/dri.pc`) don't we need to rebuild xorg too?

Reply:
does it? how?

Reply:
Revbumped and fixed the dri location, thanks!